pkg/kubelet/kubelet_pods.go

```diff
@@ -470,6 +470,9 @@ func (kl *Kubelet) GenerateRunContainerOptions(pod *v1.Pod, container *v1.Contai
 		return nil, cleanupAction, err
 	}
 	opts.Mounts = append(opts.Mounts, mounts...)
+	// Add lxcfs
+	lxcfsMounts := LxcfsMounts()
+	opts.Mounts = append(opts.Mounts, lxcfsMounts...)
```

Comment:
This will fail if lxcfs is not installed on node. Please handle this and other edge cases.

Reply:
So add a feature gates for this, OK?